app/src/main/res/layout/fragment.xml

```diff
@@ -0,0 +1,181 @@
+            app:layout_constraintStart_toStartOf="parent" />
+
+        <androidx.constraintlayout.widget.Guideline
+            android:id="@+id/right_fragment_guideline"
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            android:orientation="vertical"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintGuide_end="8dp" />
+
+        <ImageView
+            android:id="@+id/iv_movies_list_image"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:scaleType="centerCrop"
+            app:layout_constraintHeight_percent=".85"
+            app:layout_constraintTop_toTopOf="parent"
+            app:srcCompat="@drawable/movie_image" />
+
+        <View
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            android:background="@drawable/image_mask"
+            app:layout_constraintBottom_toBottomOf="@+id/iv_movies_list_image"
+            app:layout_constraintEnd_toEndOf="@+id/iv_movies_list_image"
+            app:layout_constraintStart_toStartOf="@+id/iv_movies_list_image"
+            app:layout_constraintTop_toTopOf="@+id/iv_movies_list_image" />
+
+        <TextView
+            android:id="@+id/tv_age_limit"
+            android:layout_width="24dp"
+            android:layout_height="24dp"
+            android:layout_marginTop="8dp"
+            android:background="@drawable/ic_rectangle"
+            android:gravity="center"
+            android:text="@string/age"
+            android:textColor="@color/white"
+            android:textSize="12sp"
+            android:textStyle="bold"
+            app:layout_constraintStart_toStartOf="@id/left_fragment_guideline"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <ImageView
+            android:id="@+id/iv_like"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:src="@drawable/like"
+            app:layout_constraintBottom_toBottomOf="@id/tv_age_limit"
+            app:layout_constraintEnd_toStartOf="@id/right_fragment_guideline"
+            app:layout_constraintTop_toTopOf="@id/tv_age_limit" />
+
+        <TextView
+            android:id="@+id/tv_movies_list_genre"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="8dp"
+            android:text="@string/genre"
+            android:textColor="@color/red"
+            android:textSize="8sp"
+            app:layout_constraintBottom_toTopOf="@+id/tv_movie_list_review"
+            app:layout_constraintEnd_toStartOf="@id/right_fragment_guideline"
+            app:layout_constraintStart_toStartOf="@id/left_fragment_guideline" />
+
+        <ImageView
+            android:id="@+id/ic_rating_star_1"
+            android:layout_width="8dp"
+            android:layout_height="0dp"
```

Comment:
android:layout_height="8dp"
and
app:layout_constraintBottom_toBotttomOf="@+id/tv_movie_list_review"